web-common/src/features/tables/TableEntry.svelte

```diff
@@ -0,0 +1,122 @@
+<script lang="ts">
+  import { page } from "$app/stores";
+  import ContextButton from "@rilldata/web-common/components/column-profile/ContextButton.svelte";
+  import * as DropdownMenu from "@rilldata/web-common/components/dropdown-menu/";
+  import MoreHorizontal from "@rilldata/web-common/components/icons/MoreHorizontal.svelte";
+  import TableIcon from "../../components/icons/TableIcon.svelte";
+  import Tooltip from "../../components/tooltip/Tooltip.svelte";
+  import TooltipContent from "../../components/tooltip/TooltipContent.svelte";
+  import TableMenuItems from "./TableMenuItems.svelte";
+  import TableSchema from "./TableSchema.svelte";
+  import UnsupportedTypesIndicator from "./UnsupportedTypesIndicator.svelte";
+  import {
+    makeFullyQualifiedTableName,
+    makeTablePreviewHref,
+  } from "./olap-config";
+
+  export let connectorInstanceId: string;
+  export let connector: string;
+  export let database: string = ""; // The backend interprets an empty string as the default database
+  export let databaseSchema: string = ""; // The backend interprets an empty string as the default schema
+  export let table: string;
+  export let hasUnsupportedDataTypes: boolean;
+
+  let contextMenuOpen = false;
+  let showSchema = false;
+
+  $: fullyQualifiedTableName = makeFullyQualifiedTableName(
+    connector,
+    database,
+    databaseSchema,
+    table,
+  );
+  $: href = makeTablePreviewHref(connector, database, databaseSchema, table);
+  $: open = $page.url.pathname === href;
+</script>
+
+<li aria-label={fullyQualifiedTableName} class="entry group" class:open>
+  <div class="entry-header">
+    <TableIcon size="14px" className="shrink-0 text-gray-400" />
+    <Tooltip alignment="start" location="right" distance={8}>
+      <button
+        class="clickable-text"
+        on:click={() => (showSchema = !showSchema)}
+      >
+        <span class="truncate">
+          {fullyQualifiedTableName}
+        </span>
+      </button>
+      <TooltipContent slot="tooltip-content">
+        {showSchema ? "Hide schema" : "Show schema"}
+      </TooltipContent>
+    </Tooltip>
+    {#if hasUnsupportedDataTypes}
+      <UnsupportedTypesIndicator
+        instanceId={connectorInstanceId}
+        {connector}
+        {database}
+        {databaseSchema}
+        {table}
+      />
+    {/if}
+    <div class="flex-grow" />
+    <DropdownMenu.Root bind:open={contextMenuOpen}>
+      <DropdownMenu.Trigger asChild let:builder>
+        <ContextButton
+          id="more-actions-{name}"
```

Comment:
Is this (and the one below) supposed to be fullyQualifiedTableName?

Reply:
Yeah, for now. Next iteration of this will add a folder structure to represent/organize the database > database schema > table hierarchy.

Reply:
Misread. PR here: https://github.com/rilldata/rill/pull/4711